### PR TITLE
Mass accept SDDs

### PR DESCRIPTION
**Pull request by tobru**
All these SDDs have been discussed in the PRs and are now in state accepted.

### CODE REVIEW

**Review by srueg — APPROVED** [review]

**Review by corvus-ch — APPROVED** [review]

File: docs/modules/SDDs/pages/0006-commodore-configuration-catalog-compilation.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 :sdd_owner:     Simon Gerber
 :sdd_reviewers:
 :sdd_date:      2019-10-18
-:sdd_status:    draft
+:sdd_status:    accepted
```
Comment (simu) [inline docs/modules/SDDs/pages/0006-commodore-configuration-catalog-compilation.adoc]: In my opinion, Commodore implements everything described in this SDD. Shouldn't we mark the SDD as "implemented" in that case?

The [SDD Definition](https://syn.tools/syn/SDDs/0001-sdd-definition.html) is not clear about the SDD procedure when changes are made to a tool described by an SDD in state "accepted" or "implemented". What's the process when making changes to a tool or process which implements (parts of) an accepted/implemented SDD?

File: docs/modules/SDDs/pages/0007-lieutenant-management-api.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@
 :sdd_owner:     SIG Syn
 :sdd_reviewers:
 :sdd_date:      2019-10-18
-:sdd_status:    draft
+:sdd_status:    accepted
```
Comment (simu) [inline docs/modules/SDDs/pages/0007-lieutenant-management-api.adoc]: What's the status of the Lieutenant implementation compared to the SDD? Can we mark this as "implemented"?

File: docs/modules/SDDs/pages/0009-steward-cluster-agent.adoc [hunk 3]
```diff
@@ -4,7 +4,7 @@
 :sdd_owner:     Simon Rüegg
 :sdd_reviewers: 
 :sdd_date:      2019-10-18
-:sdd_status:    draft
+:sdd_status:    accepted
```
Comment (simu) [inline docs/modules/SDDs/pages/0009-steward-cluster-agent.adoc]: How does the Steward implementation match up with this SDD? Could this SDD be considered to be marked "implemented"?

File: docs/modules/SDDs/pages/0019-gitops.adoc [hunk 4]
```diff
@@ -4,7 +4,7 @@
 :sdd_owner:
 :sdd_reviewers:
 :sdd_date:      2019-11-20
-:sdd_status:    speculative
+:sdd_status:    accepted
```
Comment (simu) [inline docs/modules/SDDs/pages/0019-gitops.adoc]: Does the current ArgoCD setup cover everything here? Should we mark this as "implemented"?

**Review by simu — COMMENTED** [review]
WDYT about marking some of the SDDs as "implemented"?

At least Commodore matches the goals and design described in SDD 0006 well enough to consider the SDD "implemented" in my opinion.

File: docs/modules/SDDs/pages/0006-commodore-configuration-catalog-compilation.adoc (hunk 1 above)
Comment (tobru) [reply docs/modules/SDDs/pages/0006-commodore-configuration-catalog-compilation.adoc]: You're right, I completely forgot about the "implemented" state. This certainly makes sense here. IMHO SDDs capture a point-in-time when this was discussed. It isn't the status-quo documentation, it helps for understanding how something was "invented". The actual status-quo should be documented alongside the tool (which isn't really done today). For changing something a new SDD should be written. If something actually doesn't apply anymore or has been replaced by a new concept (SDD!) then an SDD should be marked "obsolete".

File: docs/modules/SDDs/pages/0019-gitops.adoc (hunk 4 above)
Comment (tobru) [reply docs/modules/SDDs/pages/0019-gitops.adoc]: yes

File: docs/modules/SDDs/pages/0009-steward-cluster-agent.adoc (hunk 3 above)
Comment (tobru) [reply docs/modules/SDDs/pages/0009-steward-cluster-agent.adoc]: yes

File: docs/modules/SDDs/pages/0007-lieutenant-management-api.adoc (hunk 2 above)
Comment (tobru) [reply docs/modules/SDDs/pages/0007-lieutenant-management-api.adoc]: yes

**Comment by tobru** [comment]
Updated SDD 0006, 0007, 0009 and 0019 to be marked as "implemented"

**Review by simu — APPROVED** [review]

**Pull request merged**